modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,57 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Enabling monitoring of event listeners for Triggers for user-defined projects
+
+As a cluster administrator, to gather event listener metrics for Triggers in a user-defined project and display them in the {product-title} web console, you can create a service monitor for each event listener. On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`.
+
```

Comment:
I think it should be "triggers", lowercase? 
see https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
When I reviewed the `op*.adoc` files in openshift-docs, I noticed that "Triggers" was the dominant usage. It seems to be a short name for "Tekton Triggers," which is "a Kubernetes Custom Resource Definition (CRD) controller."

Reply:
- The instances of "Triggers" in this document all seem to be proper nouns that reference this controller service, not `Trigger` resources. 
- Changing "Triggers" to lower-case might cause confusion. A reader might interpret "triggers" as something that triggers an event, not as the controller or resource.
Therefore, I recommend leaving "Triggers" unchanged.
@abrennan89 WDYT?